### ISSUE TITLE
Not working 2022

### ISSUE THREAD

**Issue by 0rbianta** [JakeFrostyYT/Pocohunter-chroot#4]
Automatically shutdowns when I try to open it.
Problem with shell forking?

**Comment by JakeFrosty** [comment]
This project has been halted for 2 years, There's a new version coming but lighter (minimal version) with a rehauled system,
I will also be making different distro for arm64 chroots like Ubuntu and Debian

as for the ETA, it depends how busy i will be for school

I can try to help out if you can give me information like what shell you use, if nethunter scripts are properly installed and any additional information that might be helpful

**Comment by 0rbianta** [comment]
I installed Net Hunter chroot over your GitHub releases and setup it with latest NetHunter app that be able to set up and execute other chroots without any problem. It unpacked and set up your chroot to /data/local/nhsystem as excepted but when I try to access it over nethunter terminal, it failed and app closed itself like forked shell failed to run. I dont neet nethunter anymore too and removed it because I made my own Android system be able to execute linux binaries. Your chroot issue is likely shell fork failing. Might be debuggable by entering the chroot and trying to execute shell and then checking the error.
